build-system/tasks/helpers.js

```diff
@@ -130,7 +130,7 @@ async function bootstrapThirdPartyFrames(options) {
       };
       watch(frameObject.max).on(
         'change',
-        debounce(watchFunc, watchDebounceDelay)
+        debounce(watchFunc, watchDebounceDelay, {immediate: true})
```

Comment:
From looking at the `debounce` [api docs](https://github.com/component/debounce#api), it looks like the third param is a boolean and not an object. So I believe this needs to be:

```suggestion
        debounce(watchFunc, watchDebounceDelay, /* immediate */ true)
```

Reply:
You're right! Interesting that it just happened to work anyway, likely because an object is truthy